docs/application/vstools/Tizen/import.md

```diff
@@ -0,0 +1,25 @@
+# Importing Tizen Studio projects into Visual Studio
+
+The following sections explain how to use Visual Studio Extension for Tizen to import your projects created in Tizen Studio to Visual Studio.
+
+##### NOTE: 
+Ensure that the Tizen Studio project was exported to CLI before importing to Visual Studio
```

Comment:
Tizen Custom Markdown Format should be used for notes.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning
```suggestion
> [!NOTE]
> Ensure that the Tizen Studio project was exported to CLI before importing to Visual Studio
```

Reply:
Thanks @safir-srbd. Adding these changes to new patch